src/editors/containers/VideoEditor/index.test.tsx

```diff
@@ -1,49 +1,88 @@
-import 'CourseAuthoring/editors/setupEditorTest';
-import { shallow } from '@edx/react-unit-test-utils';
-
-import { formatMessage } from '../../testUtils';
+import { render, initializeMocks } from '@src/testUtils';
 import VideoEditor from '.';
+import * as hooks from './hooks';
+
+const reduxSelectors = require('../../data/redux').selectors;
 
 jest.mock('../EditorContainer', () => 'EditorContainer');
 jest.mock('./components/VideoEditorModal', () => 'VideoEditorModal');
 
-jest.mock('./hooks', () => ({
-  ErrorContext: {
-    Provider: 'ErrorContext.Provider',
-  },
-  errorsHook: jest.fn(() => ({
-    error: 'hooks.errorsHook.error',
-    validateEntry: jest.fn().mockName('validateEntry'),
-  })),
-  fetchVideoContent: jest.fn().mockName('fetchVideoContent'),
-}));
-
-jest.mock('../../data/redux', () => ({
-  selectors: {
-    requests: {
-      isFinished: jest.fn((state, params) => ({ isFailed: { state, params } })),
-    },
-    app: {
-      isLibrary: jest.fn(state => ({ isLibrary: state })),
-    },
-  },
-}));
-
-jest.mock('@openedx/paragon', () => ({
-  ...jest.requireActual('@openedx/paragon'),
-  Spinner: 'Spinner',
-}));
-
 describe('VideoEditor', () => {
   const props = {
     onClose: jest.fn().mockName('props.onClose'),
-    intl: { formatMessage },
-    studioViewFinished: false,
-    isLibrary: false,
+    returnFunction: jest.fn().mockName('props.returnFunction'),
   };
-  describe('snapshots', () => {
-    test('renders as expected with default behavior', () => {
-      expect(shallow(<VideoEditor {...props} />).snapshot).toMatchSnapshot();
+  beforeEach(() => {
+    initializeMocks();
+  });
```

Comment:
```suggestion
```

I don't think we should have `initializeMocks()` called in both the outer `beforeEach` and the inner `beforeEach` (line 35) - I believe it will be called twice, and will be inefficient. This call can be removed.